react/StoreLink.tsx

```diff
@@ -24,10 +35,25 @@ export default function StoreLink(props: Props) {
     setShouldReplaceUrl(Boolean(modalDispatch))
   }, [modalDispatch])
 
-  return (
+  return target && target !== '_self' ? (
+    <a target={target} href={href} className={handles.link}>
+      {label && <span className={handles.label}>{label}</span>}
+      {hasChildren(children) && (
+        <div className={handles.childrenContainer}>{children}</div>
+      )}
+    </a>
+  ) : (
     <Link to={href} className={handles.link} replace={shouldReplaceUrl}>
       {label && <span className={handles.label}>{label}</span>}
-      {children && <div className={handles.childrenContainer}>{children}</div>}
+      {hasChildren(children) && (
+        <div className={handles.childrenContainer}>{children}</div>
+      )}
     </Link>
   )
 }
+
+StoreLink.schema = {
+  title: 'Link',
+}
```

Comment:
Don't forget to do this using `messages`

Reply:
I think I shouldn't translate `Link`